packages/circuit-ui/components/ImageInput/ImageInput.docs.mdx

```diff
@@ -49,3 +49,24 @@ There are some caveats with this approach:
 ### Using a div element
 
 <Story id="forms-imageinput--custom-component-div" />
+
+## Accessibility
+
+### Best practices
+
+#### Error message associated with input field
+
+Error messages should be simple and clear to understand, also provide guidance on how to correct mistakes.
+The error messages must be associated with respective controls (e.g. Input field), via `aria-describedby` or labeling.
+The screen readers announce the error message with a live region by applying an `aria-live` property.
+
+The `aria-describedby` is added to the Input field, and `aria-invalid="true"` when the error is triggered.
+The `aria-live="polite"` indicates that the error message will be announced after completing other announcements first.
+
+### Resources
+
+#### Related articles
+
+- [Testing error messages](https://www.davidmacd.com/blog/test-aria-describedby-errormessage-aria-live.html) Web Accessibility Training
+- [Using live regions to identify error messages](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19.html)
+- [Error messages with aria-live] (https://gaurav5430.medium.com/accessibility-quick-wins-error-messages-with-aria-live-7a622cb606f9)
```

Comment:
These accessibility best practices are covered by the component already. I think including them here in the docs might confuse developers and make them believe that they need to take care of them.

Reply:
You mean the whole section under the #### Error message associated with input field? Your recommendation is to remove all of it?

Reply:
Yes. I remember @robinmetral and you having a similar discussion a while back — whether these internal accessibility requirements should be documented in Storybook. Can either of you recall the result of that conversation?

Reply:
Removed the section in the docs for now, and lets sync on later then for the general docs approach for the Accessibility.